.github/workflows/release.yaml

```diff
@@ -163,38 +163,38 @@ jobs:
           signing-password: ${{ secrets.COSIGN_PASSWORD }}
 
-  build-gcr-deployer:
-    name: Build GCR deployer image
-    environment: Release
-    runs-on: ubuntu-latest
-    needs: [ prepare ]
-    env:
-      IMAGE: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}:${{ needs.prepare.outputs.version }}
-      IMAGE_NO_PREFIX: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}:${{ needs.prepare.outputs.version_without_prefix }}
-    steps:
-    - name: Checkout
-      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-    - name: Login to Docker Hub
-      uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
-      with:
-        registry: gcr.io
-        username: ${{ secrets.GCR_USERNAME }}
-        password: ${{ secrets.GCR_JSON_KEY }}
-    - name: Set up QEMU
-      uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
-    - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # 3.10.0
-    - name: Build and push
-      uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83 # 6.18.0
-      with:
-        platforms: ${{ env.DEPLOYER_PLATFORMS }}
-        provenance: false
-        context: ./config/helm
-        file: ./config/helm/Dockerfile
-        push: true
-        tags: ${{ env.IMAGE }},${{ env.IMAGE_NO_PREFIX }}
-        annotations: |
-         ${{ env.GOOGLE_MARKETPLACE_ANNOTATION }}
+  # build-gcr-deployer:
+  #     name: Build GCR deployer image
+  #     environment: Release
+  #     runs-on: ubuntu-latest
+  #     needs: [ prepare ]
+  #     env:
+  #       IMAGE: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}:${{ needs.prepare.outputs.version }}
+  #       IMAGE_NO_PREFIX: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}:${{ needs.prepare.outputs.version_without_prefix }}
+  #     steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+  #     - name: Login to Docker Hub
+  #       uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+  #       with:
+  #         registry: gcr.io
+  #         username: ${{ secrets.GCR_USERNAME }}
+  #         password: ${{ secrets.GCR_JSON_KEY }}
+  #     - name: Set up QEMU
+  #       uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
+  #     - name: Set up Docker Buildx
+  #       uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # 3.10.0
+  #     - name: Build and push
+  #       uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83 # 6.18.0
+  #       with:
+  #         platforms: ${{ env.DEPLOYER_PLATFORMS }}
+  #         provenance: false
+  #         context: ./config/helm
+  #         file: ./config/helm/Dockerfile
+  #         push: true
+  #         tags: ${{ env.IMAGE }},${{ env.IMAGE_NO_PREFIX }}
+  #         annotations: |
+  #          ${{ env.GOOGLE_MARKETPLACE_ANNOTATION }}
```

Comment:
I think we should just remove this code (for now) or provide context as to why it's commented out (maybe something like "moved to gcr-deployer-image.yaml as per DAQ-11982")

Reply:
good idea, I added a comment to give some context https://github.com/Dynatrace/dynatrace-operator/pull/5246/commits/69426a9fe81fb6cb5cba6e6b600b950834fddcc8